app/models/manageiq/providers/cloud_automation_manager/configuration_manager/refresher.rb

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
This one's for you @Fryguy ;)

Reply:
:laughing: I was really hoping this would be the only commit in the PR :joy: :joy: